sds_data_manager/stacks/api_gateway_stack.py

```diff
@@ -86,7 +88,40 @@ def __init__(
             http_method = lambda_info["httpMethod"]
 
             # Define the API Gateway Resources
-            resource = api.root.add_resource(route)
+            resource = self.api.root.add_resource(route)
 
             # Create a new method that is linked to the Lambda function
             resource.add_method(http_method, apigw.LambdaIntegration(lambda_fn))
+
+    def subscribe_to_sns(self, sns_topic: aws_sns.Topic):
+        # Define the metric the alarm is based on
+        # List of Metric options for API Gateway:
+        # https://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-metrics-and-dimensions.html
+        metric = cw.Metric(
+            namespace="AWS/ApiGateway",
+            metric_name="Latency",
+            dimensions_map={"ApiName": self.api.rest_api_name},
+            period=Duration.minutes(1),
+            statistic="Maximum",
+        )
+
+        # Define the alarm
+        cw_alarm = cw.Alarm(
```

Comment:
```suggestion
        cloudwatch_alarm = cw.Alarm(
```
Unnecessarily abbreviated variable names is a pet peeve of mine, but maybe this one gets a pass if `cw` is just the standard way to name things like this?

Reply:
Agreed, thanks for bringing this up. I updated this and the imports. I think we could actually remove a bunch of the `from aws_cdk import aws_xxx as xxx` and just change them to `from aws_cdk import aws_xxx`, the `aws_xxx` is actually somewhat helpful for disambiguating resources we create vs import from cdk.